src/views/tls/tls-version/index.js

```diff
@@ -4,45 +4,88 @@ import {CardContent} from '@material-ui/core';
 import CardHeader from '@material-ui/core/CardHeader';
 import ChartContainer from 'common/chart-container';
 import CircularProgress from '@material-ui/core/CircularProgress';
+import MenuItem from '@material-ui/core/MenuItem';
 import MuiAlert from '@material-ui/lab/Alert';
 import Paper from '@material-ui/core/Paper';
 import {ResponsivePie} from '@nivo/pie';
 import Snackbar from '@material-ui/core/Snackbar';
 import Tab from '@material-ui/core/Tab';
 import Tabs from '@material-ui/core/Tabs';
 import React, {useEffect, useState} from 'react';
+// eslint-disable-next-line no-use-before-define
```

Comment:
Is there a reason to skip the ESLint rule here? There are very few cases when it's needed, and I think here is not. Lmk if you need help.